.github/workflows/ci.yaml

```diff
@@ -179,6 +179,35 @@ jobs:
         npm test
         npm run system-test
 
+  combined-lib-test-esm:
+    needs: build
+    runs-on: ubuntu-latest
+    container: gcr.io/gapic-images/googleapis:20250404
```

Comment:
I see this container has a date - will it get automatically updated by our tools?